homeassistant/components/discovery/__init__.py

```diff
@@ -146,8 +146,8 @@
     for platform in enabled_platforms:
         if platform in DEFAULT_ENABLED:
             logger.warning(
-                'Deprecated config for discovery: service %s is enabled by'
-                'default - ignoring',
+                "Please remove %s from your discovery.enable configuration "
+                "as it is now enabled by default.",
```

Comment:
We don't end logging messages with period.

Reply:
Gah, got me again. Fixed.